src/main/java/underdogs/devbie/question/controller/QuestionController.java

```diff
@@ -0,0 +1,92 @@
+package underdogs.devbie.question.controller;
+
+import java.net.URI;
+
+import javax.validation.Valid;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+import underdogs.devbie.auth.controller.resolver.LoginUser;
+import underdogs.devbie.question.dto.QuestionCreateRequest;
+import underdogs.devbie.question.dto.QuestionResponse;
+import underdogs.devbie.question.dto.QuestionResponses;
+import underdogs.devbie.question.dto.QuestionUpdateRequest;
+import underdogs.devbie.question.service.QuestionService;
+import underdogs.devbie.user.domain.User;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/questions")
+public class QuestionController {
+
+    private final QuestionService questionService;
+
+    @PostMapping
+    public ResponseEntity<Void> save(
+        @LoginUser User user,
+        @Valid @RequestBody QuestionCreateRequest request) {
+        Long questionId = questionService.save(user.getId(), request);
+        return ResponseEntity
+            .created(URI.create(String.format("/api/questions/%d", questionId)))
+            .build();
+    }
+
+    @GetMapping
+    public ResponseEntity<QuestionResponses> readAll() {
+        QuestionResponses responses = questionService.readAll();
+        return ResponseEntity
+            .ok()
+            .body(responses);
```

Comment:
```suggestion
        return ResponseEntity.ok(responses)
```
별건 아니지만 어떻게할지 다같이 정해볼까요?

Reply:
앨런네는 ok(responses)로 하신건가요?
body에 내려보낸다는 의미가 명확해서 일부러 body로 사용하긴했는데, 
ok안 에 넣는것도 코드가 심플하고 괜찮은것 같네요!

팀에서 합의가 이뤄지면 수정하도록 하겠습니다.